python/tk_desktop2/websockets/websockets_server.py

```diff
@@ -207,7 +207,7 @@ def _new_connection(self, socket_id, name, address, port, request):
 
         # The origin coming from the request's raw header will be a bytearray. We're
         # going to want it as a string, so we'll go ahead and convert it right away.
-        origin_site = request.rawHeader(QtCore.QByteArray(six.b("origin"))).data()
+        origin_site = request.rawHeader(six.b("origin")).data()
```

Comment:
I meant `b"origin"` in the other PR. No need for `six.b`. The `b` prefix in 2.7 is a no-op and in Python 3 is a byte-string.